composer.json

```diff
@@ -16,5 +15,11 @@
   "support": {
     "issues": "https://github.com/Automattic/WPCOM-Legacy-Redirector/issues",
     "source": "https://github.com/Automattic/WPCOM-Legacy-Redirector"
+  },
+  "require-dev": {
+    "automattic/vipwpcs": "^2.2",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
+    "phpunit/phpunit": "^7"
   }
 }
+
```

Comment:
Don't include the `composer.lock` file in this package, as that prescribes what versions of dependencies are working for just your set-up - someone with a different PHP version may need other versions of dependencies.